### ISSUE TITLE
[management] Gateway context path issues 

### ISSUE THREAD

**Issue by Subbu0117** [gravitee-io/issues#3275]
I did one test api in API 1.30.5 and its successfully saved but when I tried to edit the gateway context path its showing blank actually it has data in DB.

Also I tried to edit context path but says saved but showing blank in UI as well as logs. 

### Expected Behavior
create new API context path to be showed for view and edit screen.

### Current Behavior
Gateway context path showing blank all the time. 

### Possible Solution
Looks some JS issue in updating. 

### Steps to Reproduce (for bugs)
1. installed new API Mgmt, API gateway, UI
2. create one test api
3. try edit the created API, you will see gateway context blank
4. save new context will also show blank.

### Context
while editing context path

### Your Environment

* Version used: 1.30.5
* Browser Name and version: chrome 75
* Operating System and version: redhat

**Comment by brasseld** [comment]
Can you reproduce on https://demo.gravitee.io ?

**Comment by Subbu0117** [comment]
Thank you for your response. 

I created echo api here  https://demo.gravitee.io  and it works without an issue. I hope APIM version is 1.30.5

Changes I did in my environment are
copied gravitee.ymls from 1.28.0 to 1.30.5
and constant.json from 1.28.0 and 1.30.5
with new MongoDB and Elasticsearch 6.8
Gravitee gateway, APIM and UI started with no issues and created echo, but gatewway context path is blank  and gateway instance not showing in APIM.
